sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_job_property.py

```diff
@@ -0,0 +1,81 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict
+
+from azure.ai.ml._schema._deployment.batch.batch_job_property import BatchJobPropertySchema
+from azure.ai.ml.entities._deployment.deployment_settings import BatchRetrySettings
+from azure.ai.ml.entities._deployment.compute_binding import ComputeBinding
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+from azure.ai.ml.entities._system_data import SystemData
+from azure.ai.ml._restclient.v2020_09_01_dataplanepreview.models import BatchJob as BatchJobData
+
+
+class BatchJobProperty:
+    """Batch Job Property entity
+
+    :param copmute: Compute configuration used to set instance count.
+    :type id: ComputeBinding
+    :param dataset: Input dataset.
+    :type dataset: str
+    :param error_threshold: Error threshold, if the error count for the entire input goes above
+         this value, the batch inference will be aborted. Range is [-1, int.MaxValue] -1 value 
+         indicates, ignore all failures during batch inference.
+    :type error_threshold: int
+    :param input_data:Input data for the job.
+    :type input_date: Dict[str, JobInput]
```

Comment:
```suggestion
    :type input_data: Dict[str, JobInput]
```

Also, do we need to update line 46 to include the types too?